### ISSUE TITLE
Error in eigenvalue analysis with new build

### ISSUE THREAD

**Issue by e-dub** [KratosMultiphysics/Kratos#2959]
After a new build of Kratos, I am getting an error from the eigenvalue analysis that I cannot wrap my head around. 

I saw that there were some changes to the JSON files and implemented these.  

Further, I am getting the same error when trying to setup an eigenvalue problem in GiD.

Any ideas for me?  The error message is below.

Thanks in advance!

```
  File "MainKratos.py", line 20, in <module>
    simulation.Run()
  File "~/kratos/kratos/python_scripts/analysis_stage.py", line 47, in Run
    self.Initialize()
  File "~/kratos/kratos/python_scripts/analysis_stage.py", line 77, in Initialize
    for process in self._GetListOfProcesses():
  File "~/kratos/kratos/python_scripts/analysis_stage.py", line 204, in _GetListOfProcesses
    self._list_of_processes = self._CreateProcesses("processes", order_processes_initialization)
  File "~/kratos/applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py", line 88, in _CreateProcesses
    list_of_processes = super(StructuralMechanicsAnalysis, self)._CreateProcesses(parameter_name, initialization_order)
  File "~/kratos/kratos/python_scripts/analysis_stage.py", line 252, in _CreateProcesses
    list_of_processes += factory.ConstructListOfProcesses(value) # Does this work? or should it be processes[name]
  File "~/kratos/kratos/python_scripts/process_factory.py", line 18, in ConstructListOfProcesses
    p = python_module.Factory(item, self.Model)
  File "~/kratos/applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py", line 16, in Factory
    return PostProcessEigenvaluesProcess(Model, settings["Parameters"])
  File "~/kratos/applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py", line 39, in __init__
    self.model_part = Model[settings["computing_model_part_name"].GetString()]
RuntimeError: Error: The ModelPart named : "computing_domain" was not found as root-ModelPart. The total input string was "computing_domain"


in kratos/containers/model.cpp:126:ModelPart& Model::GetModelPart(const string&)
   kratos/containers/model.cpp:133:ModelPart& Model::GetModelPart(const string&)
```

**Comment by philbucher** [comment]
hi @e-dub 
there is currently some changes going o regarding how to access the `Model` (which is the container of `ModelParts`), see #2417 and https://github.com/KratosMultiphysics/GiDInterface/issues/419

In short, to uniquely access the ModelPart you want, you should give the full name to the Model, i.e. in your case this would be `Structure.computing_domain`
=> can you try `self.model_part = Model["Structure.computing_domain"]` and tell me if it works?

**Comment by e-dub** [comment]
Thank you for the fast response!

I tried unsuccessfully by adding the suggested code as follows:

```
...
...
class StructuralMechanicsAnalysisWithEigenPostProcessing(StructuralMechanicsAnalysis):
    def Initialize(self):
        super(StructuralMechanicsAnalysisWithEigenPostProcessing,
              self).Initialize()
        self.model_part = km.Model["Structure.computing_domain"]
...
...
```
The error message remains the same. 

I have included a cleaned Python file as well as the JSON file I am using (with txt extension added). 

[Eigen_cleaned.py.txt](https://github.com/KratosMultiphysics/Kratos/files/2433314/Eigen_cleaned.py.txt)
[EigenvalueParameters.json.txt](https://github.com/KratosMultiphysics/Kratos/files/2433315/EigenvalueParameters.json.txt)

**Comment by philbucher** [comment]
I pushed a fix, see #2960 
In the meantime you can get it working with adding 
`"computing_model_part_name"    : "Structure.computing_domain",` to the parameters of the EigenPostProcess in the *.json
